src/pages/App.tsx

```diff
@@ -64,6 +66,21 @@ const Marginer = styled.div`
 `
 
 export default function App() {
+  const history = useHistory()
+
+  const ScrollToTop = () => {
```

Comment:
Does this need to be wrapped in a functional component? Because this isn't presentational, I think it would be best implemented as a hook (ie remove lines 71, 80-82, 99).

Reply:
Ah you're right it's unnecessary. Removed in the latest commit.